script.js

```diff
@@ -67,3 +67,27 @@ popOutMenu4.addEventListener('click', () => {
   const imgCross = document.querySelector('.cross-img');
   imgCross.classList.remove('active');
 });
+const popInProjectDetails = document.querySelector('.button-1');
```

Comment:
- [ ] Please make sure that the JavaScript code implements an array of objects to store JavaScript objects containing the information for each project. Also, each object contains at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code. Kindly create an array to store the projects' details.